diagnostic_rt_qpcr/operation_types/run_qpcr/protocol.rb

```diff
@@ -39,20 +44,29 @@ def default_job_params
   #     input of type JSON and named `Options`.
   #
   def default_operation_params
-    {}
+    {
+      thermocycler_model: 'BioRad CFX96' # TestThermocycler::MODEL,
+    }
   end
 
   ########## MAIN ##########
 
   def main
     setup_test_options(operations: operations) if debug
 
-    @job_params = update_job_params(
+    @job_params = update_all_params(
       operations: operations,
-      default_job_params: default_job_params
+      default_job_params: default_job_params,
+      default_operation_params: default_operation_params
     )
     return {} if operations.errored.any?
 
+    available_thermocyclers = get_available_thermocyclers
+
+    remove_unavailable_operations(available_thermocyclers)
```

Comment:
Rename this method to something like `pair_ops_and_thermocyclers`. It's not clear why an operation would be unavailable.

Reply:
Then maybe follow it with `remove_unpaired_operations` or something like that.